lib/src/canonicalized_map.dart

```diff
@@ -69,6 +69,22 @@ class CanonicalizedMap<C, K, V> implements Map<K, V> {
     other.forEach((key, value) => this[key] = value);
   }
 
+  @override
```

Comment:
This package do not use `@override`, so no need to add it here, or elsewhere.
That also voids the need for the "ignore:" below.